src/components/Configuration.tsx

```diff
@@ -13,7 +13,7 @@ import "./Configuration.scss";
 import PresetSelection from './PresetSelection';
 
 export interface ConfigurationProps {
-    readonly height: number;
```

Comment:
This is wrong. We do not use numbers in our codebase, but strings like `100px` and `80vw`